src/main/java/uk/gov/hmcts/ccd/v2/external/domain/CaseDocumentMetadata.java

```diff
@@ -1,10 +1,17 @@
 package uk.gov.hmcts.ccd.v2.external.domain;
 
+import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
 
-@Data
 @Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
+@Getter
```

Comment:
@Data was causing problems in the mapping of the response to certain objects.
So I replaced this with the four annotations above.